app/models/comment.rb

```diff
@@ -1,4 +1,7 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :review
+  scope :load_reply_comment, ->(reply_id){where(reply_id: reply_id)}
+  scope :load_comment, ->{where(reply_id: nil)}
```

Comment:
bỏ ngoặc của where line 4 - 5 luôn

`reply_id: nil` trong sql có `is null` mà, sao ko dùng hì